node/src/main/scala/coop/rchain/node/api/grpc.scala

```diff
@@ -46,7 +44,7 @@ object GrpcServer {
     }
 
   def acquireExternalServer[
-      F[_]: Capture: Monad: MultiParentCasperRef: Log: SafetyOracle: BlockStore: Futurable](
+      F[_]: Sync: Capture: Monad: MultiParentCasperRef: Log: SafetyOracle: BlockStore: Futurable](
```

Comment:
`Sync`/`Monad` redundancy. Also `Sync`/`Capture` redundancy.

Reply:
Sync/Capture is redundant?

Reply:
`Sync.delay` is the effectively the same as `Capture.capture`.

Reply:
Sure but we can't remove capture until we remove atomic monad state